packages/tools/client-debugger/client-debugger-view/src/components/FluidHandleView.tsx

```diff
@@ -79,8 +84,11 @@ export function FluidHandleView(props: FluidHandleViewProps): React.ReactElement
 
 	return (
 		<Accordion
-			key={containerId}
-			header={<div>{`${visualTree.metadata}, ${visualTree.nodeKind}`}</div>}
+			header={
+				<div>{`${visualTree.metadata && visualTree.metadata} : ${
```

Comment:
Prettier makes the weirdest choices sometimes 🤣

Reply:
True! I first thought I added a line by a mistake 😆!